lib/plugins/aws/package/compile/events/sqs/index.js

```diff
@@ -45,10 +46,13 @@ class AwsCompileSQSEvents {
             let EventSourceArn;
             let BatchSize = 10;
             let Enabled = true;
+            let MaximumBatchingWindowInSeconds = 0;
```

Comment:
Let's not set the default. If `event.sqs.maximumBatchingWindowInSeconds` is not set, let's not set property at all.

Reply:
Not sure whether this is the best way to do it: https://github.com/serverless/serverless/commit/d70c58b49c8a30ec194102801727f1b570db37d0#diff-bb7849521252332fd089024a0031c78cbc85ac2eda8c9d4df916582f865f9a76R99-R105

Let me know if you have a more elegant way.